components/prism-elixir.js

```diff
@@ -1,4 +1,8 @@
 Prism.languages.elixir = {
+	'doc': {
+    pattern: /@(?:doc|moduledoc)\s+(?:("""|''')(?:\\[\s\S]|(?!\1)[^\\])+\1|.*)/,
```

Comment:
The `.*` matches all characters until end of line in this example: `@doc here goes some documentation`.
If there's a better way please let me know and I'll change it

Reply:
I apologize for my incredibly vague question.

I wanted to ask what the `.*` is supposed to tokenize. I'm not very familiar with Elixir, so I don't know what can and can't be done with attributes. I know that `.*` matches everything until the end of the line but is that correct?

It will correctly match all examples [here](https://hexdocs.pm/elixir/writing-documentation.html) and [here](https://elixir-lang.org/getting-started/module-attributes.html) but I wonder if that's enough.

For `@doc here goes some documentation`: How is `here goes some documentation` parsed? From what the documentation says, it sounds like it has to be a valid value, so is something like this possible?

```elixir
@doc deprecated: """Use Foo.bar/2 instead.
This function was removed for one reason or another.
"""
@doc since: "1.0.0" # some comment
```